src/main/kotlin/no/nav/familie/ef/sak/behandling/dto/BehandlingDto.kt

```diff
@@ -43,6 +43,22 @@ fun Behandling.tilDto(stønadstype: StønadType, vedtaksdato: LocalDateTime? = n
             ?: (if (this.status == BehandlingStatus.FERDIGSTILT) this.sporbar.endret.endretTid else null)
     )
 
+fun List<Behandling>.tilDto(stønadstype: StønadType) = this.map {
```

Comment:
Tror vi kan bruke `fun Behandling.tilDto(stønadstype: StønadType, vedtaksdato: LocalDateTime? = null): BehandlingDto =` i stedet for denne